src/main/java/com/github/dockerjava/core/util/CertificateUtils.java

```diff
@@ -41,115 +49,157 @@ public static boolean verifyCertificatesExist(String dockerCertPath) {
         return result;
     }
 
+    /**
+     * @param dockerCertPath with standard named files.
+     */
     public static KeyStore createKeyStore(final String dockerCertPath) throws NoSuchAlgorithmException,
             InvalidKeySpecException, IOException, CertificateException, KeyStoreException {
-        KeyPair keyPair = loadPrivateKey(dockerCertPath);
-        List<Certificate> privateCertificates = loadCertificates(dockerCertPath);
+        return createKeyStore("key.pem", "cert.pem");
```

Comment:
i guess it should just load files into string, or pass streams and everything would work